synfig-studio/src/gui/resources/ui/studio_menubar.xml

```diff
@@ -172,10 +175,12 @@
 				<item>
 					<attribute name="label" translatable="yes">_Input Devices...</attribute>
 					<attribute name="action">win.input-devices</attribute>
+                    <attribute name="icon">input_devices_icon</attribute>
 				</item>
 				<item>
 					<attribute name="label" translatable="yes">_Preferences...</attribute>
 					<attribute name="action">app.setup</attribute>
+                    <attribute name="icon">preferences_icon</attribute>
```

Comment:
wrong indentation

Reply:
I fixed them all by using tab indentation instead of space ones. The whole file had a mix of them all so I turned into the one that it seems you use in the rest of the project. I hope. it's fine like this.